osu.Game.Tests/Visual/Gameplay/TestSceneHUDOverlay.cs

```diff
@@ -31,8 +33,14 @@ public class TestSceneHUDOverlay : OsuManualInputManagerTestScene
         private Drawable hideTarget => hudOverlay.KeyCounter;
         private FillFlowContainer<KeyCounter> keyCounterFlow => hudOverlay.KeyCounter.ChildrenOfType<FillFlowContainer<KeyCounter>>().First();
 
-        [Resolved]
-        private OsuConfigManager config { get; set; }
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            Dependencies.Cache(localConfig = new OsuConfigManager(LocalStorage));
+        }
+
+        [SetUpSteps]
+        public void SetUp() => Schedule(() => localConfig.SetValue(OsuSetting.HUDVisibilityMode, HUDVisibilityMode.Always));
```

Comment:
And I believe the previous issue is related to this. This should be `SetUp`, not `SetUpSteps`.

Reply:
Ah, that make sense.